### PR TITLE
Write document about vector db 

### PR DESCRIPTION
**Pull request by zozond**
### Motivation

I don't know about vector db. 
So, I want to learn about vector db.

### Modifications

add vector db markdown document 

### Result

add vector db document

### CODE REVIEW

**Review by TheWeaVer — COMMENTED** [review]
Thank you for contribution, please write all contents in English 😄

File: hephaistus/sub-theme/vectordb/VectorDB.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# VectorDB
```
[the remark this answers is not in the document]
Comment (zozond) [reply hephaistus/sub-theme/vectordb/VectorDB.md:1]: OK, I'll move this document!

File: hephaistus/main-theme/distribution-system/chord/playground/zozond/chord-impl/settings.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+pluginManagement {
```
Comment (nullist0) [inline hephaistus/main-theme/distribution-system/chord/playground/zozond/chord-impl/settings.gradle.kts:1]: This is not related with the vector db. Please remove this file and make a new PR if it is needed.

**Comment by TheWeaVer** [comment]
If this document includes ACID and CAS, then it would be more informative document 😄

File: hephaistus/main-theme/distribution-system/chord/playground/zozond/chord-impl/settings.gradle.kts (hunk 2 above)
Comment (zozond) [reply hephaistus/main-theme/distribution-system/chord/playground/zozond/chord-impl/settings.gradle.kts:1]: OK, I'll remove it.

**Comment by TheWeaVer** [comment]
Thank you! Could revise file name to it? 🙂 
- introduce-vector-db.md
- introduce-vector-db(kor).md

**Comment by zozond** [comment]
Reflected your recommendation. 
@TheWeaVer

**Comment by TheWeaVer** [comment]
Please rename file also 😄 
* vector-db-introduce.md
* vector-db-introduce(kor).md

**Comment by zozond** [comment]
File will be located 1 depth folder.

**Review by TheWeaVer — APPROVED** [review]
LGTM
